package.json

```diff
@@ -8,8 +8,17 @@
     "start": "next start",
     "lint": "next lint",
     "prepare": "husky install",
+    "test": "jest --watch",
     "cypress:open": "cypress open",
-    "cypress:run": "cypress run"
+    "cypress:run": "cypress run",
+    "test:coverage": "rm -f ./.nyc_output/out.json && yarn test --coverage --watchAll=false && yarn cypress:run && node ./scripts/coverage-merge.js && nyc report",
+    "test:coverage-merge": "node ./scripts/coverage-merge.js"
+  },
+  "jest": {
+    "collectCoverage": true,
+    "coverageReporters": [
+      "json"
+    ]
```

Comment:
Can we move this to `jest.config.js`? I think it causes this warning when running the test 🤔 

https://user-images.githubusercontent.com/1772999/159621080-70ec5a58-b6a3-4e68-822a-204b52a6bb38.mov

Reply:
sure, done in https://github.com/tyrro/react-survey/pull/49